pom.xml

```diff
@@ -284,6 +292,20 @@
 			<artifactId>powermock-api-mockito2</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>appframework-api</artifactId>
+			<version>${appframeworkVersion}</version>
+			<scope>provided</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>coreapps-omod</artifactId>
+			<version>1.28.0</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
Do we need these here or can we put these in the dependency management section instead?

Reply:
Also, do we even actually need these?

Reply:
@ibacher I think we need these because we are using `BreadcrumbHelper` and `AppDescriptor` in the controller class

Reply:
That makes some kind of sense, but I think in that case:

1. They should go in the dependency management section so only the OMOD depends on them.
2. We should modify the config.xml to ensure that those are noted as required modules for this module.